src/graph.ts

```diff
@@ -203,13 +211,33 @@ export class GraphBuilder {
             if (resource.Type === 'AWS::EC2::VPC') {
                 this.vpcNodes[node.logicalId] = { vpcNode: node, vpcCidrBlockNode: undefined };
             }
+        } else if (node.construct.constructInfo?.fqn === 'aws-cdk-lib.CfnResource') {
```

Comment:
Previously we were missing resources that were not in the tree (e.g. nodes with id `Default`).
I noticed this because CDK CustomResources label the `CfnCustomResource` with ID default.

We need https://github.com/pulumi/pulumi-cdk/issues/186 to validate all resources of the template get mapped.